Creational/README.rst

```diff
@@ -6,7 +7,7 @@ suitable to the situation. The basic form of object creation could result in
 design problems or added complexity to the design. Creational design patterns
 solve this problem by somehow controlling this object creation.
 
-* [AbstractFactory](AbstractFactory) [:notebook:](http://en.wikipedia.org/wiki/Abstract_factory_pattern)
+* :doc:`AbstractFactory <AbstractFactory/README>` `Wikipedia <http://en.wikipedia.org/wiki/Abstract_factory_pattern>`_
```

Comment:
This works well locally, but on **Read The Docs**, it does not work. I will investigate more.

http://designpatternsphp.readthedocs.org/en/latest/README/